app.ts

```diff
@@ -51,10 +63,11 @@ app.get('/docs.json', (req: Request, res: Response) => {
     res.setHeader('Content-Type', 'application/json');
     res.send(swaggerSpec);
 });
-
+ 
 app.use(express.static(path.join(__dirname,'public')))
 
 const schemaOptions = {
+    customJsStr: Analytics ? Analytics.swaggerRegistration(JSON.parse(process.env[Analytics.name])):undefined,
```

Comment:
Can this have a better name so we know that it is used for the plausible file

Reply:
At the moment the plausible provider calls this - however it might be that other providers (such as fathom) might also register itself for JS integration.

Reply:
@ageddesi  - I'm open to ideas?  `CustomJsStr` is aproperty provided by Swagger - so we can't rename that - but the function `swaggerRegistration`  - could be renamed to whatever works best - got any ideas?

Reply:
Maybe we just leave it for now then :)

Reply:
cool - but if you (or someone else thinks of a better name then we should change ir)